src/App/Fossa/VPS/Scan/RunWiggins.hs

```diff
@@ -32,12 +34,24 @@ data WigginsOpts = WigginsOpts
   , spectrometerArgs :: [Text]
   }
 
-generateWigginsOpts :: Path Abs Dir -> Severity -> ProjectRevision -> ScanType -> FilterExpressions -> ApiOpts -> ProjectMetadata -> WigginsOpts
-generateWigginsOpts scanDir logSeverity projectRevision scanType fileFilters apiOpts metadata =
-  WigginsOpts scanDir (generateSpectrometerArgs logSeverity projectRevision scanType fileFilters apiOpts metadata)
+generateWigginsScanOpts :: Path Abs Dir -> Severity -> ProjectRevision -> ScanType -> FilterExpressions -> ApiOpts -> ProjectMetadata -> WigginsOpts
+generateWigginsScanOpts scanDir logSeverity projectRevision scanType fileFilters apiOpts metadata =
+  WigginsOpts scanDir (generateSpectrometerScanArgs logSeverity projectRevision scanType fileFilters apiOpts metadata)
```

Comment:
[optional]: For long parenthetical sections, I **personally** prefer using `$` instead, though that's not a hard-and-fast rule.
```suggestion
  WigginsOpts scanDir $ generateSpectrometerScanArgs logSeverity projectRevision scanType fileFilters apiOpts metadata
```